tests/nnfw_api/src/ValidationTestPipelineSession.cc

```diff
@@ -73,6 +69,148 @@ TEST_F(ValidationTestPipelineSession, pipeline_session_test_model)
   remove("./partition_map.json");
 }
 
+TEST_F(ValidationTestPipelineSession, prepare_pipeline_001)
+{
+  std::ifstream readFile("./partition_map.json");
+
+  if (readFile.good())
+  {
+    remove("./partition_map.json");
+  }
+
+  NNFW_ENSURE_SUCCESS(nnfw_create_session(&_session));
+  NNFW_ENSURE_SUCCESS(nnfw_load_model_from_modelfile(
+    _session, NNPackages::get().getModelAbsoluteFilePath("mobilenet_v1_1.0_224").c_str()));
+  ASSERT_EQ(nnfw_prepare_pipeline(_session, "./partition_map.json"), NNFW_STATUS_ERROR);
+}
+
+TEST_F(ValidationTestPipelineSession, prepare_pipeline_002)
+{
+  build_partition_map();
+
+  NNFW_ENSURE_SUCCESS(nnfw_create_session(&_session));
+  NNFW_ENSURE_SUCCESS(nnfw_load_model_from_modelfile(
+    _session, NNPackages::get().getModelAbsoluteFilePath("mobilenet_v1_1.0_224").c_str()));
+  NNFW_ENSURE_SUCCESS(nnfw_prepare_pipeline(_session, "./partition_map.json"));
+
+  remove("./partition_map.json");
```

Comment:
`build_partition_map` also invoke `remove` function.
do we have to invoke this function 2 times in each test ?

Reply:
I missed it. Thank you for your review.
I removed this function.